SPECS/dotnet-runtime-3.1/dotnet-runtime-3.1.spec

```diff
@@ -1,31 +1,39 @@
 Summary:        Microsoft .NET Core Runtime
 Name:           dotnet-runtime-3.1
 Version:        3.1.5
-Release:        1%{?dist}
+Release:        2%{?dist}
+License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-License:        MIT
-Url:            https://github.com/dotnet/core
 Group:          Development/Tools
-ExclusiveArch:  x86_64
+URL:            https://github.com/dotnet/core
 Source0:        https://download.visualstudio.microsoft.com/download/pr/d00eaeea-6d7b-4e73-9d96-c0234ed3b665/0d25d9d1aeaebdeef01d15370d5cd22b/dotnet-runtime-3.1.5-linux-x64.tar.gz
-
+Source1:        https://download.visualstudio.microsoft.com/download/pr/6827d794-a218-4352-b3b3-a19ec773c975/e3e53bc2f20df220a29c6e09f74d8a00/aspnetcore-runtime-3.1.5-linux-x64.tar.gz
+Requires:       glibc
 Requires:       icu
-Requires:       libunwind
 Requires:       krb5
+Requires:       libgcc
+Requires:       libstdc++
+Requires:       libunwind
 Requires:       lttng-ust
 Requires:       openssl
 Requires:       zlib
-Requires:       glibc
-Requires:       libgcc
-Requires:       libstdc++
+ExclusiveArch:  x86_64
 
 %description
 .NET Core is a development platform that you can use to build command-line
 applications, microservices and modern websites.
 
+%package -n aspnetcore-runtime-3.1
```

Comment:
Why `aspnetcore-runtime-3.1` and not `aspnetcore-runtime-%{version}`?

Also, why does it have to be a subpackage of this one instead of a separate new spec? It doesn't seem to share any build or installation steps with the default one. I'm not sure what is common between them.

Reply:
Going to leave it as explicitly aspnetcore-runtime-3.1 since the full version is 3.1.5, but the "-runtime-3.1" corresponds with our existing dotnet-runtime versioning scheme.

Good point on the subpackage part. I've split aspnetcore out into a separate spec and re-tested build/installation of both.